js/src/personal/kazi/Kazi.page.tsx

```diff
@@ -3,6 +3,7 @@ import styles from './Kazi.module.css'
 import { DarkModeToggle } from './components/DarkModeToggle/DarkModeToggle'
```

Comment:
nit: Linear id in commit message

Reply:
Added linear id and test to commit message